change/@microsoft-teams-js-6ffe0681-75ea-4146-bdbd-31fd134ce89a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I haven't looked at beachball in this repo in a while so I'm not sure about this- but would this be a "none" change since the unit tests don't affect the content of the package? Although, the change in communication.ts might be a small bug fix- if so that would be nice to mention in the comment field

Reply:
I think patch is correct given that there's a small change to communication.uninitializeCommunication. I will update the changefile to make a note of that